packages/elements/src/utils/oas/index.ts

```diff
@@ -134,7 +134,9 @@ function computeChildNodes(
 function findMapMatch(key: string | number, map: ISourceNodeMap[]): ISourceNodeMap | void {
   if (typeof key === 'number') return;
   for (const entry of map) {
-    if (!!entry.match?.match(key) || (entry.notMatch !== void 0 && !entry.notMatch.match(key))) {
+    const escapedKey = key.replace(/([()[{*+.$^\\|?])/g, '\\$1');
```

Comment:
The capturing group is a bit redundant in this case

```suggestion
    const escapedKey = key.replace(/[\\^$*+?.()|[\]{}]/g, '\\$&')
```

Reply:
Ah makes sense. Tested your suggestion locally and it still appears to behave as expected (not throwing). 

Only I am not 100% sure that this change can somehow cause regression but if I understand the code correctly the `entry.match()` and `entry.notMatch()` function are checking if some requirements defined are met